egs/ami/s5/local/nnet3/prepare_parallel_datadirs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
```

Comment:
at the top of scripts like this, a couple of lines saying what it's for would be nice.